.github/workflows/htmlvalidator.yml

```diff
@@ -0,0 +1,17 @@
+name: HTML Check
+
+on:
+  pull_request:
+    paths:
+      - '**.htm*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v5 # Requried will all actions
+    - name: Checks HTML5
+      uses: Cyb3r-Jak3/html5validator-action@v8.0.0
```

Comment:
To improve security it would probably be good to pin all actions to the specific release commit, similar to how it's done in the releng.aggregator workflows.